package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "cbioportal-frontend",
   "private": true,
-  "version": "3.2.13-alpha.0",
+  "version": "3.2.14-alpha.0",
```

Comment:
better to update this to `3.2.13-alpha.1`, though it is not a big deal since we never publish this package.